integration/ct_prep_test.sh

```diff
@@ -62,20 +37,28 @@ done
 # some kind of sequencer mastership election.
 pushd "${TRILLIAN_ROOT}" > /dev/null
 declare -a RPC_SERVER_PIDS
-for port in ${RPC_PORTS}; do
-    echo "Starting Log RPC server on port ${port}"
-    ./trillian_log_server --private_key_password=towel --private_key_file=${TESTDATA}/log-rpc-server.privkey.pem --port ${port} --sequencer_sleep_between_runs="1s" --batch_size=500 --export_metrics=false &
-    pid=$!
-    RPC_SERVER_PIDS+=(${pid})
-done
-popd > /dev/null
+base=${BASE_RPC_PORT}
+for ((i=0; i < RPC_SERVER_COUNT; i++)); do
+  base=$((base + 2))
+  port=$(pickUnusedPort ${base})
+  if [[ $i -eq 0 ]]; then
+    RPC_PORTS="${port}"
+    RPC_SERVERS="localhost:${port}"
+  else
+    RPC_PORTS="${RPC_PORTS} ${port}"
```

Comment:
Suggestion: you could always use the "else" branch logic and trim the first character at the end of the loop

```bash
for (( ... )); do
  ...
done
RPC_PORTS="${RPC_PORTS:1}"
RPC_SERVERS="${RPC_SERVERS:1}"
```

Ditto below.